### PR TITLE
resized breadcrumb

### PR DESCRIPTION
**Pull request by thomascraig**
resolves #1138 updated h1 to use h1 component and resized typography.

### CODE REVIEW

File: components/guided_experience.js [hunk 1]
```diff
@@ -37,12 +39,15 @@ const nextButton = css`
...
+  font-size: 14px !important;
+  line-height: 21px;
+  width: auto;
 `;
 
 const jumpButton = css`
-  font-size: 1.5em;
+  font-size: 14px !important;
+  line-height: 21px;
+  color: #006cc9;
```
Comment (sastels) [inline components/guided_experience.js]: you can use `theme.colour.cerulean` for this colour

**Comment by sastels** [comment]
Can you get the breadcrumbs to be closer to "My selection"? From https://app.zeplin.io/project/5acbd659dc553ec84805afd4/screen/5b758b9388a3e676eae08241

image

File: components/guided_experience.js (hunk 1 above)
Comment (thomascraig) [reply components/guided_experience.js]: ok, so all the colors in zeplin are added to the globalTheme?

**Review by sastels — APPROVED** [review]
minor comments, 🦑

File: components/guided_experience.js (hunk 1 above)
Comment (sastels) [reply components/guided_experience.js]: yeah, I added that yesterday I think?

Comment (thomascraig) [reply components/guided_experience.js]: @sastels should I replace all the color ref's from hex vs globalTheme within the file?
eg: background-color: #39824d !important;   used on the next button

Comment (sastels) [reply components/guided_experience.js]: yeah, any ones you find in `theme.js`. `#39824d` is `fernGreen` :)

**Comment by thomascraig** [comment]
@sastels I decreased the grid size from 3 to 2, will try size 1 but thinking it will be too small.

Attached is with grid size from 2 to 1, get some wrapping happening due to size restraints.
screencapture-localhost-3000-guided-2018-09-06-16_26_25

**Comment by sastels** [comment]
ah fair enough. Maybe just leave it as it was before, we can always open a new issue to look at it later.

**Comment by thomascraig** [comment]
@sastels I can tweak it more to look like the design, will just need to utilize more css.

**Comment by sastels** [comment]
up to you 😄

**Pull request merged**